euclidean_gcd/euclidean_gcd.c

```diff
@@ -1,10 +1,10 @@
 #include<stdio.h>
```

Comment:
We can do this in a separate patch as this PR was not responsible for it. 
Also, I am think we can create some regex checks for these kinds of issues.